databricks/sdk/core.py

```diff
@@ -253,32 +253,69 @@ def refresh(self) -> Token:
 class AzureCliTokenSource(CliTokenSource):
     """ Obtain the token granted by `az login` CLI command """
 
-    def __init__(self, resource: str):
+    def __init__(self, resource: str, subscription: str = ""):
         cmd = ["az", "account", "get-access-token", "--resource", resource, "--output", "json"]
+        if subscription != "":
+            cmd.append("--subscription")
+            cmd.append(subscription)
```

Comment:
nit: we can use `extend` if we are adding more than one item. It would be more readable.